vignettes/articles-online-only/opencl.Rmd

```diff
@@ -43,6 +43,21 @@ OpenCL in CmdStanR, an OpenCL runtime for the target device must be installed.
 A guide for the most common devices is available in the CmdStan manual's
 [chapter on parallelization](https://mc-stan.org/docs/2_26/cmdstan-guide/parallelization.html#opencl).
 
+In case of using Windows, CmdStan requires the `OpenCL.lib` to compile the model.
+If you experience issue compiling the model with OpenCL, run the below script and set
+`path_to_opencl_lib` to the path to the `OpenCL.lib` file on your system.
+If you are using CUDA, the path should be similar to the one listed here.
+
+```{r windows-opencl, message=FALSE, results='hide'}
+path_to_opencl_lib <- "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.5/lib/x64"
+cpp_options = list(
+  paste0("LDFLAGS+= -L\"",path_to_opencl_lib,"\" -lOpenCL")
+)
+
+cmdstanr::cmdstan_make_local(cpp_options = cpp_options)
+cmdstanr::rebuild_cmdstan()
+```
```

Comment:
This looks good, just one question: is the idea for this chunk to be evaluated or is it enough to just include the lines of code here and not actually rebuild cmdstan when we generate the vignette (in which case we could set `eval=FALSE`)?

Reply:
Yeah, this is just a sample that the users on windows need to run once. It should only be in the vignette, but not actually run.

Reply:
Ok I think in that case we should just set `eval=FALSE` in the chunk, otherwise it will be run when generating the vignette: 

```suggestion
```{r windows-opencl, eval=FALSE}
path_to_opencl_lib <- "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.5/lib/x64"
cpp_options = list(
  paste0("LDFLAGS+= -L\"",path_to_opencl_lib,"\" -lOpenCL")
)

cmdstanr::cmdstan_make_local(cpp_options = cpp_options)
cmdstanr::rebuild_cmdstan()
```
```